Source/IGListAutoSectionController.m

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "IGListAutoSectionController.h"
+
+#import <IGListKit/IGListAssert.h>
+#import <IGListKit/IGListDiffable.h>
+#import <IGListKit/IGListDiff.h>
+#import <IGListKit/IGListBindable.h>
+
+typedef NS_ENUM(NSInteger, IGListAutoSectionState) {
+    IGListAutoSectionStateIdle = 0,
+    IGListAutoSectionStateUpdateQueued,
+    IGListAutoSectionStateUpdateApplied
+};
+
+@interface IGListAutoSectionController()
+
+@property (nonatomic, strong) id object;
+@property (nonatomic, strong) NSArray<id<IGListDiffable>> *viewModels;
```

Comment:
Yes! Absolutely we should. I can imagine plenty of cases where someone would need to grab the view model.